pom.xml

```diff
@@ -101,19 +110,33 @@
         <dependency>
             <groupId>org.jenkins-ci.plugins</groupId>
             <artifactId>git</artifactId>
-            <version>3.4.0</version>
+            <version>3.9.3</version>
+            <exclusions>
```

Comment:
If you have to do that, it is rather suppressing the issue. Better to declare dependencies directly in pom.xml, especially workflow-step-api which is used in production and excluded below

Reply:
Upon reflection, I think, in the end, I ended up specifying these dependencies directly anyhow.  I'll remove the exclusions and update the pom accordingly.

Reply:
They were not needed, I backed them out.  Unfortunately the result of iterative work and not reviewing my PR well enough.